_posts/2020-07-14-mill-brook-summer-2020.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Mill Brook Summer 2020
+image: assets/images/millbrooksummer20201.png
+---
+As we all try and adjust to the changes each day brings, the beauty of nature always awaits us outside. Here are some photos of our community gardens, wildlife, and the Path’s calming vistas. For everyone’s safety and protection, please wear a mask while out walking. Thank you.
+![aLt tExT?](https://github.com/millbrook/millbrook.github.io/assets/images/millbrooksummer20202.png "MillBrookSummer2020")
```

Comment:
- [ ] as requested in previous comment, please add more descriptive alt text. `aLt tExT` will not make search engines to better index the site
- [ ] in addition, also as requested in the previous comment, please use proper jekyll template for the asset url. last comment had a basic example. [docs here](https://jekyllrb.com/docs/liquid/filters/). 
- [ ] finally markdown syntax here is wrong for including images. see any markdown cheatsheet for examples. I had an example in the previous comments. This is what it will render like now:
  <img width="1316" alt="image" src="https://user-images.githubusercontent.com/932940/87634382-23f44480-c70b-11ea-85f4-0497b306280c.png">